androbot/tests.py

```diff
@@ -282,3 +287,19 @@ def test_start_test_after_reset_session():
     assert Actions().get_next_test(user.tg_user_id).text_question == question.text_question
     Actions().remove_user(user)
     Actions().remove_questions("test")
+
+
+def test_add_bot_score():
+    user = TelegramUser(
+        tg_user_id=Utils.get_random_number(5),
+        name=Utils.get_random_text(10),
+        username=Utils.get_random_text(10),
+        specialty="test",
+    )
+    Actions().add_user(user)
```

Comment:
Тут всё можно объединить под один with
```suggestion
    with Actions() as act:
        act.add_user(user)
```
И остальные тесты все же поправить бы